flowdock-text.js

```diff
@@ -963,25 +963,29 @@ if (typeof FlowdockText === "undefined" || FlowdockText === null) {
     };
   };
 
-  FlowdockText.mentionsAllTags = ["@everyone", "@everybody", "@all", "@anyone", "@anybody"];
```

Comment:
I don't see any point of removing this easily accessible list of the tag definitions.